corporate/tests/test_stripe.py

```diff
@@ -1344,14 +1340,15 @@ def test_replace_payment_source(self, *mocks: Mock) -> None:
         self.assertEqual(2, RealmAuditLog.objects.filter(
             event_type=RealmAuditLog.STRIPE_CARD_CHANGED).count())
 
-    @patch("corporate.lib.stripe.billing_logger.info")
-    def test_downgrade(self, mock_: Mock) -> None:
+    def test_downgrade(self) -> None:
         user = self.example_user("hamlet")
         self.login_user(user)
         with patch("corporate.lib.stripe.timezone_now", return_value=self.now):
             self.local_upgrade(self.seat_count, True, CustomerPlan.ANNUAL, 'token')
-        response = self.client_post("/json/billing/plan/change",
-                                    {'status': CustomerPlan.DOWNGRADE_AT_END_OF_CYCLE})
+        with self.assertLogs('corporate.stripe', 'INFO') as m:
+            response = self.client_post("/json/billing/plan/change",
+                                        {'status': CustomerPlan.DOWNGRADE_AT_END_OF_CYCLE})
+            self.assertRegexpMatches(m.output[0], r"INFO:corporate.stripe:Change plan status: Customer.id: \d*, CustomerPlan.id: \d*, status: \d*")
```

Comment:
Check the exact status. The value of status is fixed unlike Customer.id and CustomerPlan.id. See `status` in CustomerPlan model.